frontend-react/src/components/Table/Pagination.test.tsx

```diff
@@ -9,17 +9,12 @@ import Pagination, {
 } from "./Pagination";
 
 describe("Pagination", () => {
-    // Text match for getting pagination elements by role to capture previous
-    // and next links, page numbers, and overflow indicators (which are for
-    // presentation).
-    const ITEM_ROLE = /listitem|presentation/i;
-
     test.each([
         {
             description: "on the first page of an unbounded set",
             slots: [1, 2, 3, 4, 5, 6, OVERFLOW_INDICATOR],
             currentPageNum: 1,
-            expectedItems: ["1", "2", "3", "4", "5", "6", "…", "Next"],
+            expectedItems: ["1", "2", "3", "4", "5", "6", "Next", "…"],
```

Comment:
Huhhhhh...interesting that the wrong order didn't cause any issues with the textContent comparison below 🤔

Reply:
That's due to the items array being [...listitems, presentation (if ...)]. the regex that was previously used doesn't seem to be allowed anymore for the query so I had to grab the two role types manually and adjust expectedItems to fit the new order